Design/design - element web setup with HTTPS only.txt

```diff
@@ -43,11 +43,13 @@ Build and deploy
 E:
 cd \dev\github\pegacorn-communicate-app-web       (or where you have cloned the git project)
 
-docker build --rm --build-arg IMAGE_BUILD_TIMESTAMP="%date% %time%" -t pegacorn-communicate-web:1.0.0-snapshot --file Dockerfile.element .
-OR do a full build with 
-  1) Copy paste docker-ssl-certificate from E:\docker-ssl-cert to your pegacorn-communicate-app-web directory
-  2) Copy pegacorn-communicate-matrix-react-sdk and pegacorn-communicate-matrix-js-sdk into src/ directory
-  3) docker build --rm --build-arg IMAGE_BUILD_TIMESTAMP="%date% %time%" -t pegacorn-communicate-web:1.0.0-snapshot .
+Do a full build with 
+  1) copy /Y \docker-ssl-certs\PRDPKICA.crt .
+  2) Xcopy /E /I /Y ..\pegacorn-communicate-matrix-js-sdk pegacorn-communicate-matrix-js-sdk
```

Comment:
Guideline says /e to be used with //s and /t but I guess I is going to remove need of other two I have mentioned as it copies subdirectories by default.

Reply:
This was based on https://www.windows-commandline.com/copy-directory-command-line/ and tested, ok on my DEV Azure Workstation